package.json

```diff
@@ -92,6 +92,7 @@
   },
   "devDependencies": {
     "@build-tracker/cli": "^1.0.0-beta.15",
+    "@rollup/plugin-commonjs": "^20.0.0",
```

Comment:
the package moved, we were using the old version. newer version necessary to fix a bug that appeared from building in the original pr